doc/nrf/libraries/security/nrf_security/doc/driver_config.rst

```diff
@@ -592,7 +608,9 @@ The following table shows ECC curve support for each driver:
 | FRP256v1                 | Not supported             | Not supported              | Not supported             |
 +--------------------------+---------------------------+----------------------------+---------------------------+
 
-The option :kconfig:option:`CONFIG_PSA_USE_CC3XX_KEY_MANAGEMENT_DRIVER` enables the :ref:`nrf_security_drivers_cc3xx` for key management using ECC curves.
+The Kconfig option :kconfig:option:`CONFIG_PSA_USE_CC3XX_KEY_MANAGEMENT_DRIVER` enables the :ref:`nrf_security_drivers_cc3xx` for key management using ECC curves.
+
+The Kconfig option :kconfig:option:`CONFIG_PSA_USE_CRACEN_KEY_MANAGEMENT_DRIVER` enables the :ref:`nrf_security_drivers_cracen` for key management using ECC curves.
 
 RNG configurations
```

Comment:
I see there is CONFIG_PSA_USE_CRACEN_CTR_DRBG_DRIVER, so how should that be described here?